### PR TITLE
docs: ssr should now work

### PR DESCRIPTION
**Pull request by rikbrowning**

### CODE REVIEW

File: packages/@featherds/autocomplete/src/components/FeatherAutocomplete.spec.js [hunk 1]
```diff
@@ -196,6 +196,8 @@ const baseFunctionality = (type) => {
       });
     });
     describe("Accessibility", () => {
+      beforeEach(() => jest.setTimeout(60000));
```
Comment (jmillar-nanthealth) [inline packages/@featherds/autocomplete/src/components/FeatherAutocomplete.spec.js:199]: Is this really required!?

**Review by jmillar-nanthealth — COMMENTED** [review]
Looking good overall save for the comment on the test setup

**Comment by rikbrowning** [comment]
I hit an issue locally on my laptop were it took ages to complete. It seems
to run much faster in CI

**Comment by jmillar-nanthealth** [comment]
Merging manually

**Pull request closed without merging**